### ISSUE TITLE
Update readme for throttling

### ISSUE THREAD

**Issue by erayerdin** [AWConant/jikanpy#21]
Current readme says:

 > Jikanpy does not make any attempts to rate limit itself, so use it as responsibly as you would use the API primitively.

However, current Jikan API documentation clearly states that [it is throttled](https://jikan.docs.apiary.io/#introduction/information/rate-limiting).

Will update readme.

**Comment by abhinavk99** [comment]
The readme is saying that Jikanpy doesn't rate limit you, not that Jikan API doesn't. I guess we could make it clearer.

**Comment by erayerdin** [comment]
Wow, it seems I couldn't catch that. Do not merge #22, I will add one more commit.

**Comment by erayerdin** [comment]
Okay, changed the paragraph to:

```
Jikanpy is a Python wrapper for [Jikan](https://github.com/jikan-me/jikan),
providing bindings for all API functionality. Because it is intended to be
pretty much identical, please consult [Jikan's
documentation](https://jikan.docs.apiary.io/#) for thornier details on how it is meant to
be used. Perhaps most importantly, Jikanpy does not make any attempts to rate
limit itself, so use it as responsibly as you would use the API primitively. However,
remember that Jikan API has limitations, check out [this section](https://jikan.docs.apiary.io/#introduction/information/rate-limiting)
of documentation in order to see to what extent the API is limited or throttled.
```

**Comment by erayerdin** [comment]
Okay, changed the section to below:

```
so use it as responsibly as you would use the API primitively and
remember that
```

**Comment by abhinavk99** [comment]
Merged #22
